<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogBlogDialog</class>
 <widget class="QDialog" name="LogBlogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="gboxTop">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout"/>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QCheckBox" name="ckUserName">
                <property name="text">
                 <string>操作员</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="cboxUserName">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>管理员</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>顾客</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
         <zorder>tableView</zorder>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QPushButton" name="btnDelete">
           <property name="cursor">
            <cursorShape>OpenHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>删除(&amp;D)</string>
           </property>
           <property name="icon">
            <iconset resource="image.qrc">
             <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btnSelect">
           <property name="cursor">
            <cursorShape>OpenHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>查询(&amp;S)</string>
           </property>
           <property name="icon">
            <iconset resource="image.qrc">
             <normaloff>:/image/select.png</normaloff>:/image/select.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnExcel">
           <property name="cursor">
            <cursorShape>OpenHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>功能开发中</string>
           </property>
           <property name="text">
            <string>导出(&amp;E)</string>
           </property>
           <property name="icon">
            <iconset resource="image.qrc">
             <normaloff>:/image/print.png</normaloff>:/image/print.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>操作日志</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="topMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
